### PR TITLE
Added more clients, wallets and tools

### PR DESCRIPTION
**Pull request by CyberDexter**
Added more web clients, plebstr for android, profile tools and apps for content creators as well as self-custodial wallets.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -138,17 +138,19 @@ There are more native clients in development, Nostros[^nostros] and Nozzle[^nozz
...
+- [iris.to](https://iris.to/) - Clean interface & rich in features
 - [nostrgram.co](https://nostrgram.co/) - Focus on images and media, supports multiple layout styles
 
-On Android, you can use the [Kiwi Browser](https://kiwibrowser.com/) which
-allows you to install Alby or nos2x, which in turn allows you to use any web
-client. [Yosup](https://yosup.app/) and [Hamstr](https://hamstr.to/) have good
-mobile experiences, for example.
+On Android, you can use:
+
+- [Plebstr](https://play.google.com/store/apps/details?id=com.plebstr.client) - Good short message client similar to Twitter.
```
Comment (dergigi) [inline index.md:151]: Plebstr should be where Amethyst is (Android clients section above)

Comment (CyberDexter) [reply index.md:151]: Agreed. if you merge this I'll fix it with the next pull request.

Comment (dergigi) [reply index.md:151]: Merged it & I've just made the change on `master`.

Thanks again for your contribution! 🙏💜

Comment (CyberDexter) [reply index.md:151]: great. I'll sync up.

**Pull request merged**